src/App.tsx

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import todos from './api/todos';
+import users from './api/users';
+import { TodoList } from './components/TodoList';
+import { Todo } from './components/TodoInfo';
+
+const preparedTodos: Todo[] = todos.map(obj => {
+  const foundUser = users.find(user => user.id === obj.userId);
+  const temp:Todo = {
+    ...obj,
+    user: foundUser === undefined ? null : foundUser,
```

Comment:
i guess this should work
```suggestion
    user: foundUser || null,
```